src/openrct2/peep/Peep.cpp

```diff
@@ -2378,7 +2380,7 @@ static void peep_update_falling(rct_peep * peep)
     peep->next_y = peep->y & 0xFFE0;
     peep->next_z = saved_map->base_height;
 
-    sint32 edx = saved_map->properties.surface.slope & TILE_ELEMENT_SLOPE_W_CORNER_DN;
+    sint32 edx = saved_map->properties.path.type & 0x7;
```

Comment:
Is there really no define or function for this? I'm trying to get rid of direct access to tile element fields.

Reply:
There are two parts combined here: `FOOTPATH_PROPERTIES_FLAG_IS_SLOPED | FOOTPATH_PROPERTIES_SLOPE_DIRECTION_MASK`

Reply:
Yeah this was a ninja edit so that I don't forget that this was wrong. I want to go through and finally demystifi next var 27